model/labels/labels_stringlabels.go

```diff
@@ -283,6 +283,12 @@ func (ls Labels) WithoutEmpty() Labels {
 	return ls
 }
 
+// ByteSize returns the approximate size of the labels in bytes.
+// String header size is ignored because it should be amortized to zero.
```

Comment:
Can we say "may be shared across multiple copies of the `Labels`" ? 
Such copies are not common in Prometheus, so I don't think this is a strong position, but I think it's more accurate.

Reply:
I updated that in 0bf0e8475d44c06bbd3476d02fd3638fb7e3ed08.